Dockerfile.e2e-s390x

```diff
@@ -0,0 +1,64 @@
+## Step 1: Build tests
+FROM s390x/golang:1.8.5-jessie as builder
```

Comment:
These are not yet multi-arch?

Reply:
It is. I don't think this needs to be changed from golang:1.8.5-alpine3.6 though since that is multiarch now